pepper-apis/dsm-core/src/main/resources/liquibase/PEPPER-873_mercury_order_fields.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="PEPPER-873_mercury_new_fields" author="pegah">
+        <addColumn tableName="ddp_mercury_sequencing">
+            <column name="order_message" type="TEXT"></column>
+            <column name="status_message" type="TEXT"></column>
+        </addColumn>
+    </changeSet>
+    <changeSet id="PEPPER-873_mercury_status_detail_changes" author="pegah">
+        <modifyDataType columnName="status_detail" newDataType="VARCHAR(2000)" tableName="ddp_mercury_sequencing"/>
```

Comment:
There are errors in dev that the field is small so I changed the size

Reply:
Why not convert the field to text?

Reply:
@otchet-broad if our theory is correct I think we will see a size increase in the messages again